pkg/sources/github/github.go

```diff
@@ -942,8 +962,7 @@ func (s *Source) addAllVisibleOrgs(ctx context.Context) {
 			} else {
 				continue
 			}
-			s.log.V(2).Info("adding organization for repository enumeration", "id", org.ID, "org", name)
-			common.AddStringSliceItem(name, &s.orgs)
```

Comment:
what replaced this deleted call?

Reply:
This was actually the main reason I did this refactor. We were setting the orgs and repos using this `common.AddStringsliceItem` function in order to keep track of repos and orgs. It was wildly inefficient given a number of reasons.
1. Since it was created as a slice we were always checking the slice before adding a new item to the slice. For some large customers we were doing a check to see if we already added the repo or org for each new repo or org. This was replaced by using the cache instead of this slice.
2. Since the slice didn't have a cap, we had to keep growing the slice for large customers and moving old entries into the new slice.
3. This was just confusing, we were doing slice ptr operations that I felt we could get rid of completely by instead using a map where we get all the benefits without all this extra overhead.